### PR TITLE
Update BSPO import

### PR DESCRIPTION
**Pull request by anitacaron**
In this updated import we have:
- removed bspo axiomes that was in `uberon-edit.obo`
- moved `in central side of` from Uberon to BSPO
- imported BSPO:0020001 that is the new `in central side of` BSPO term

These modifications are part of the Uberon cleansing to have a real uberon-base artifact. Related to https://github.com/obophenotype/uberon/issues/2453

### CODE REVIEW

File: src/ontology/imports/bspo_import.owl [hunk 1]
```diff
@@ -402,7 +454,7 @@
 
     <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/BSPO_0000120">
         <rdfs:subPropertyOf rdf:resource="http://purl.obolibrary.org/obo/BSPO_0000126"/>
-        <owl:inverseOf rdf:resource="http://purl.obolibrary.org/obo/BSPO_0000121"/>
+        <owl:propertyDisjointWith rdf:resource="http://purl.obolibrary.org/obo/BSPO_0000121"/>
```
Comment (matentzn) [inline src/ontology/imports/bspo_import.owl:457]: Expected change?

File: src/ontology/imports/bspo_import.owl [hunk 2]
```diff
@@ -656,33 +710,6 @@
     
 
 
-    <!-- http://purl.obolibrary.org/obo/BSPO_0001114 -->
```
Comment (matentzn) [inline src/ontology/imports/bspo_import.owl:659]: Not used in Uberon?

File: src/ontology/imports/bspo_import.owl [hunk 3]
```diff
@@ -882,176 +913,189 @@
...
-    <owl:Axiom>
-        <owl:annotatedSource rdf:resource="http://purl.obolibrary.org/obo/BSPO_0000054"/>
-        <owl:annotatedProperty rdf:resource="http://purl.obolibrary.org/obo/IAO_0000115"/>
-        <owl:annotatedTarget>An anatomical region bounded by a plane perpendicular to an axis through the middle.</owl:annotatedTarget>
-        <oboInOwl:hasDbXref rdf:datatype="http://www.w3.org/2001/XMLSchema#string">BSPO:cjm</oboInOwl:hasDbXref>
-    </owl:Axiom>
-    
-
-
-    <!-- http://purl.obolibrary.org/obo/BSPO_0000059 -->
```
Comment (matentzn) [inline src/ontology/imports/bspo_import.owl:927]: Not used in Uberon?

**Review by matentzn — COMMENTED** [review]
Do I see this correctly that the PR has no impact on the classification (subclass axioms)?

File: src/ontology/imports/bspo_import.owl (hunk 2 above)
Comment (anitacaron) [reply src/ontology/imports/bspo_import.owl:659]: No

File: src/ontology/imports/bspo_import.owl (hunk 3 above)
Comment (anitacaron) [reply src/ontology/imports/bspo_import.owl:927]: Not anymore, it was used in the previous uberon property `in central side of` as a reference (`see also` annotation)

**Review by matentzn — APPROVED** [review]

**Pull request merged**